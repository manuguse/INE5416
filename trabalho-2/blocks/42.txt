1 1 8 8 12 12 12 16
2 2 9 9 13 13 13 16
3 3 3 9 14 16 16 16
3 3 10 10 14 14 16 16
4 4 4 10 14 14 18 19
5 5 10 10 14 17 17 19
6 7 11 11 11 15 17 15
6 7 7 7 15 15 15 15