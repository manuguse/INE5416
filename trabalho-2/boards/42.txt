- - - - 3 1 - -
- - - - - - - -
- - 4 - - 2 5 3
1 - 2 5 - 5 - 1
- 1 - - - 4 - -
- - - - - - 2 -
- - 2 - 3 5 - 4
- - - 3 - 3 - 2