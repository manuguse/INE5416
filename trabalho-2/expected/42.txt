2 1 2 1 3 1 2 7
1 2 1 3 2 3 1 6
2 5 4 2 6 2 5 3
1 3 2 5 3 5 4 1
2 1 3 4 2 4 1 2
1 2 1 3 1 3 2 1
2 4 2 1 3 5 1 4
1 2 1 3 1 3 6 2